blob/azureblob/azureblob.go

```diff
@@ -816,11 +822,43 @@ func (b *bucket) ListPaged(ctx context.Context, opts *driver.ListOptions) (*driv
 	return page, nil
 }
 
+func (b *bucket) getDelegationCredentials(ctx context.Context) (*azblob.UserDelegationCredential, error) {
+	validity := 48 * time.Hour
```

Comment:
`validity` is not the best name here `currentTime.Add(validity)` isn't as obvious as it could be (validity implies a boolean to me)